lib/src/main/proto/bio/terra/tanagra/underlay/mapping.proto

```diff
@@ -72,8 +72,23 @@ message RelationshipMapping {
     ColumnId foreign_key = 2;
   }
 
+  // A mapping where a relationship is navigated by a separate intermediate table.
```

Comment:
Yes, it should also be possible to express such a relationship (table 1 FKs to intermediate table FKs to table 2) using two FK mappings. As you say, this would require 3 entity definitions, one for each table.

The goal here was to allow defining this relationship with just the existing 2 entities. The intermediate table doesn't really mean anything to the UI, in the same way that the other two tables do. I wanted to avoid defining extra entities just based on the SQL schema.